src/test/java/Homework17.java

```diff
@@ -0,0 +1,36 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.junit.Assert;
+
+// ...
+
```

Comment:
public class Homework17 {

Define class. Class is missing and move your all code inside the class.